haskell/test/Test/Ligo/BaseDAO/Common/StorageHelper.hs

```diff
@@ -5,7 +5,13 @@ module Test.Ligo.BaseDAO.Common.StorageHelper
   , getFullStorageView
 
   , GetTotalSupplyFn
+  , GetProposalFn
+  , GetFreezeHistoryFn
+  , GetQuorumThresholdAtCycleFn
   , getTotalSupplyEmulator
+  , getQtAtCycleEmulator
+  , getProposalEmulator
+  , getFreezeHistoryEmulator
   -- , getTotalSupplyNetwork
```

Comment:
I think it would be nice to keep these separate by "group" like it was the case for `TotalSupply` and `GetVotePermitCounter` ones before, both here and in the code below.

Reply:
Have changed